src/openzaak/conf/includes/base.py

```diff
@@ -290,12 +290,11 @@
 # Django setup configuration
 #
 SETUP_CONFIGURATION_STEPS = [
-    "openzaak.config.bootstrap.site.SiteConfigurationStep",
-    "openzaak.config.bootstrap.notifications.AuthNotificationStep",
-    "openzaak.config.bootstrap.notifications.NotificationsAPIConfigurationStep",
-    "openzaak.config.bootstrap.selectielijst.SelectielijstAPIConfigurationStep",
-    "openzaak.config.bootstrap.demo.DemoUserStep",
-    "openzaak.config.bootstrap.authorizations.AuthorizationConfigurationStep",
+    "django_setup_configuration.config.sites.steps.SitesConfigurationStep",
+    "zgw_consumers.contrib.setup_configuration.steps.ServiceConfigurationStep",
+    "notifications_api_common.contrib.setup_configuration.steps.NotificationConfigurationStep",
+    # "openzaak.config.setup_configuration.steps.SelectielijstAPIConfigurationStep",
+    # "openzaak.config.setup_configuration.steps.DemoUserStep",
```

Comment:
IDK if demo user is actually used and I think selectielijst is covered in the authorization or external PRs

Reply:
I think DemoConfig can be definetely removed since it's covered partly by `commonground-api-common.JWTSecretsConfigurationStep` and will be wrapped up in #1842